doc/source/ray-contribute/development.rst

```diff
@@ -195,17 +195,15 @@ After that, you can now move back to the top level Ray directory:
 
 .. code-block:: shell
 
-  cd ../..
+  cd -
 
 
 Now let's build Ray for Python. Make sure you activate any Python virtual (or conda) environment you could be using as described above.
 
-Enter into the ``python/`` directory inside of the Ray project directory and install the project with ``pip``:
+Install the project with ``pip``:
 
 .. code-block:: bash
 
-  # Install Ray.
-  cd python/
```

Comment:
shouldn't we be in the python directory though?

Reply:
ah yes, added back